scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -32,6 +32,9 @@ for name in toolchains/*/*/ ; do
 done
 git config submodule.toolchains/qemu.update none
 
+# Don't automatically initialize generators with big submodules (e.g. linux source)
+git config submodule.generators/sha3.update none
```

Comment:
This doesn't work as intended, it will not check out all the sha3 sources. Instead something like this should be better:

```
git config submodule.generators/sha3.update none
git submodule update --init --recursive
git config --unset submodule.generators/sha3.update
git submodule update --init generators/sha3
```

Reply:
This is causing the CI to fail.